.travis.yml

```diff
@@ -110,23 +110,28 @@ before_scripts:
   - make
   - make install
 
+build_scripts:
+  - ./autogen.sh
+  - sed -i 's/$(POFILES) $(GMOFILES) \\/$(POFILES) \\/' po/Makefile.in.in
+  - ./configure
+  - make
+
 after_scripts:
   - if [ ${BUILD_TYPE} == "autotools" ];then
```

Comment:
Here, now the `$BUILD_TYPE` is `scripts`. 
https://travis-ci.org/mate-desktop/mozo/jobs/549071586#L2192

Reply:
done